docs/tools/powershell-import.md

```diff
@@ -0,0 +1,17 @@
+# Powershell import
+
+For moving an existing repo or adding NuGet packages in bulk, this may help. It simply creates a .bat file which can be executed for import.
+
+```powershell
+Get-ChildItem -Path "C:\Temp\NewPackages" -Recurse | Where-Object {$_.Extension -eq ".nupkg"} | Select-Object FullName | ForEach-Object {

```

Comment:
I like the idea of guiding users on how to bulk import packages, but this is already possible using the following command:

```
dotnet nuget push *.nupkg -s http://myserver:5000/v3/index.json -k {apikey}
```

Let's remove this document for now, and I'll add this command into the getting started steps